front/lib/actions/mcp.ts

```diff
@@ -344,6 +346,26 @@ export class MCPActionType extends BaseAction {
         : "Successfully executed action, no output.",
     };
   }
+
+  getSId(owner: LightWorkspaceType): string {
```

Comment:
Regarding [this comment](https://github.com/dust-tt/dust/pull/12922#discussion_r2126350823) I tried but I get `A 'get' accessor cannot have parameters.` error so I guess we cannot use getter? 👶

Reply:
roger that :+1: